### PR TITLE
Reset form, add hover effect to Ad component, fix minor issues 

### LINKED ISSUE #39: FE: Reset POST form after successful creation
- [x] Reset the form upon successful submission
- [x] Add hover effect to Ad component
- [x] Use Row to manage AdDetails component layout
- [x] Add label to contact information

### PR DESCRIPTION
**Pull request by wpiao**

### CODE REVIEW

File: client/src/components/AdDetails.js [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
+          </Card>
+        </Col>
+        <Col xs={6}>
+          <Form>
+            <Form.Group controlId="formBasicEmail">
+              <Form.Label>Email</Form.Label>
+              <Form.Control type="email" value={ad.email} disabled></Form.Control>
+              <br />
+              <Form.Label>Zip Code</Form.Label>
+              <Form.Control type="zipcode" value={ad.zipCode} disabled></Form.Control>
```
Comment (raymond-sde) [inline client/src/components/AdDetails.js]: zipcode is not a valid html input type. see https://www.w3schools.com/html/html_form_input_types.asp

Comment (raymond-sde) [reply client/src/components/AdDetails.js]: i would use type="text" instead

File: client/src/components/CreateAdForm.js [hunk 2]
```diff
@@ -31,6 +30,7 @@ export const CreateAdForm = (props) => {
   const submit = (e) => {
     e.preventDefault();
     handleSubmit(ad);
+    e.target.reset();
```
Comment (raymond-sde) [inline client/src/components/CreateAdForm.js]: the form in the browser may reset with this, but does the actual state of "ad" reset?

Comment (raymond-sde) [reply client/src/components/CreateAdForm.js]: what happens if submit fails? the user's info would be cleared and would have to type it all over again before resubmitting

File: client/src/components/AdDetails.js (hunk 1 above)
Comment (wpiao) [reply client/src/components/AdDetails.js]: Change the type to "text"

File: client/src/components/CreateAdForm.js (hunk 2 above)
Comment (wpiao) [reply client/src/components/CreateAdForm.js]: Good point. I make the reset happen only after successful submission.

**Review by raymond-sde — APPROVED** [review]

**Pull request merged**